pkg/shell/index.html

```diff
@@ -7,8 +7,9 @@
     <link href="index.css" rel="stylesheet">
     <link href="../../static/branding.css" rel="stylesheet">
     <script src="../base1/cockpit.js"></script>
+    <script src="../base1/po.js"></script>
     <script src="../manifests.js"></script>
-    <script src="../*/po.js"></script>
+    <script src="po.js"></script>
```

Comment:
Meh, this caused a regression, see https://github.com/cockpit-project/starter-kit/pull/542#issuecomment-1070472816

This stops showing translated menu entries. I wonder why we don't see that for cockpit's own pages? They are all translated for me in German and French. But not for starter-kit (and presumably other third-party pages)

Reply:
I sent #17145 to fix this